packages/web-vite/src/components/LabelPickerModal.tsx

```
@@ -0,0 +1,247 @@
+      return newSet
+    })
+  }
+
+  const handleSave = async () => {
+    try {
+      const labelNames = Array.from(selectedLabels)
+
+      const freshLabels = await fetchLabels()
+
+      // Convert label names to label IDs using fresh data
+      const labelIds = freshLabels
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)